app/tests/spec/views/mixins/signup-mixin.js

```diff
@@ -125,64 +125,7 @@ define(function (require, exports, module) {
         });
       });
 
-      describe('verified account', function () {
```

Comment:
From the signup-mixin's P.O.V., there is no longer a difference between verified and unverified accounts.